CsCore/PlainNetClassLib/src/Plugins/CsCore/com/csutil/algorithms/AaaaImageCompareAaaa.cs

```diff
@@ -0,0 +1,107 @@
+﻿using StbImageSharp;
+using System;
+using System.Drawing;
+using System.Numerics;
+
+namespace com.csutil.src.Plugins.CsCore.com.csutil.algorithms {
+
+    public class AaaaImageCompareAaaa {
+
+        public AaaaImageCompareAaaa() { }
+
+        public CompareResult ImageCompare(ImageResult img1, ImageResult img2, double redScale=5) {
```

Comment:
Please add a few comments to these public methods, how to use it and what the parameters such as redScale mean